lib/jnpr/junos/utils/fs.py

```diff
@@ -264,6 +265,35 @@ def _decode(fs):
         return dict((_name(fs), _decode(fs)) for fs in rsp.xpath('filesystem'))
 
     # -------------------------------------------------------------------------
+    # directory_usage - filesystem directory usage
+    # -------------------------------------------------------------------------
+
+    def directory_usage(self, path="."):
+        """
+        Returns the directory usage, similar to the unix "du" command.
+
+        :returns: approximate directory usage, including subdirectories, in bytes
+        """
+        rsp = self._dev.rpc.get_directory_usage_information(path=path, depth="0")
```

Comment:
There's a possibility this line will raise any of the exceptions raised when invoking an RPC. That's fine, but we might want to document it.